components/application-operator/charts/application/templates/deployment.yaml

```diff
@@ -73,6 +73,8 @@ spec:
         sidecar.istio.io/inject: "true"
       labels:
         app: {{ .Release.Name }}-event-service
+        kyma-grafana: enabled
+        kyma-alerts: enabled
```

Comment:
This changes will take effect only after you bump image of `application-operator`. You might want to extract the to separate PR and bump image of operator on this one.

Reply:
You are setting those to always be `enabled`, what would happen when we use this on the cluster that has no monitoring/alerting? Parametrisation is a good practice and should be applied here.

Please use the `values.yaml` file to parametrise those, there is already `eventService.deployment` field there, you could do something like:
```
labels:
  app: ...
  kyma-grafana: {{ .Values.eventService.deployment.labels.grafana }}
  kyma-alerts: {{ .Values.eventService.deployment.labels.alerts }}
```

and then in `values.yaml`:
```
eventService:
  deployment:
    ...
    labels:
      grafana: "enabled"
      alerts: "enabled"
    ...
```

Reply:
I'll update the `application-operator version` in a following PR after this PR gets merged/image built